src/pages-and-resources/app-settings-modal/AppSettingsModal.jsx

```diff
@@ -64,6 +65,8 @@ function AppSettingsModal({
   const appInfo = useModel('courseApps', appId);
   const dispatch = useDispatch();
   const submitButtonState = updateSettingsRequestStatus === RequestStatus.IN_PROGRESS ? 'pending' : 'default';
+  const isTabletOrMobile = useMediaQuery({ query: '(max-width: 768px)' });
```

Comment:
ideally, i'd like to reuse this logic in future so I think this makes it a good candidate for moving this code in utils or its own `.js` file which could help reuse this logic in other components as well.

e.g.

```
isTabletOrMobile =  useTabletAndBelowMediaQuery()
OR
const isDesktop = useDesktopMediaQuery()
```

This code is [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).